code/modules/hydroponics/grown/tea_coffee.dm

```diff
@@ -14,6 +14,7 @@
 	icon_dead = "tea-dead"
 	genes = list(/datum/plant_gene/trait/repeated_harvest)
 	mutatelist = list(/obj/item/seeds/tea/astra)
```

Comment:
you mutate normal tea into astra tea

Reply:
While Tea Astra is what you can mutate the normal, round start Tea into, it's meant to serve as an upgrade.  Tea Astra, the mutation, produces 10% Synaptizine in addition to the normal 4% vitamins and 10% tea powder.  The problem is that Tea Aspera's code `/obj/item/seeds/tea` makes it so it produces literally nothing, whereas it seems like (and from the sources I've checked against) it should produce 4% Vitamins and 10% Tea leaves, but no Synaptizine as that's the perk of mutating it.  

Unless someone is going to claim that it's part of the design to have Tea Aspera's only 
function be to mutate it to Astera for it to be any good, it seems to just be an oversight of forgetting a line of code.

Likewise, a similar format is used for the Coffee types later in the code.
`/obj/item/seeds/coffee` (Coffee arabica) and it's mutation subset `/obj/item/seeds/coffee/robusta` (Coffee Robusta) both have their own separate reagent_adds that account for their different makeup.